code/modules/projectiles/guns/mounted.dm

```diff
@@ -1,12 +1,11 @@
 /obj/item/weapon/gun/energy/gun/advtaser/mounted
 	name = "mounted taser"
 	desc = "An arm mounted dual-mode weapon that fires electrodes and disabler shots."
-	icon_state = "armcannon"
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "taser"
+	item_state = "armcannonstun4"
```

Comment:
I added sprites that are sort of better than original ones. This has a side effect: item_state no longer displays gun's charge. But do we really need to see the charge of arm-mounted cannons by simply looking on mob? I guess not.